### ISSUE TITLE
Configuring Site Authentication - protected access

### ISSUE THREAD

**Issue by manuel-vara** [Netcentric/vg-volvotrucks-us-rd#25]
- Needed to create test content for non-released / confidential information of products

**!! If we implement this, we will lose the QG testing; to be used later, on the new product launch.**

**Comment by shirin27** [comment]
@manuel-vara is this visitor access (so login functionality on the website)?

**Comment by manuel-vara** [comment]
> @manuel-vara is this visitor access (so login functionality on the website)?

No, it's about setting up intranet access only for the team to work changes/new content.

**Comment by BeckyMedlin** [comment]
@DanielaPedrochevd @manuel-vara Is this still needed? If so, can we move to the backlog?

**Comment by BeckyMedlin** [comment]
@Gonzalo-CG @DanielaPedrochevd Discussed with @ZoeBeykirch and this is definitely needed. Please put this one in a sprint. Thank you.

**Comment by DanielaPedrochevd** [comment]
Both companies need to provide support and we don't see that feasible to happen before the end of sprint 23E- Sprint 4

**Comment by ZoeBeykirch** [comment]
Current blocker: To protect the access to Redesign website and Digital Reveal content, we would need a Volvo Azure AD admin to approve an Adobe application access to the basic profile and email of the restricted set of @volvo.com users that will be able to access. On the Volvo side, Michael Nordgren is not the admin and we are still investigating who is the person to approve this request. 

Below you can find the link and some screenshots of the Adobe related documentation:
[https://www.aem.live/docs/authentication-setup-site](https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.aem.live%2Fdocs%2Fauthentication-setup-site&data=05%7C02%7Czoe.beykirch%40volvo.com%7C2a878aef729045eae37e08dc022f0fb4%7Cf25493ae1c9841d78a330be75f5fe603%7C0%7C0%7C638387648367344542%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FrWEehd8YOEt0JkjKVdQccXGuNBiJr0287br8dgF1Ns%3D&reserved=0).
